website/events/models/event.py

```diff
@@ -178,6 +178,17 @@ class Event(models.Model, metaclass=ModelTranslateMeta):
         null=True,
     )
 
+    @property
+    def cancel_too_late_message(self):
+        return (
```

Comment:
I would put the translation in the text here instead of the view, that would make it available in the API too

Reply:
@LCKnol this is still an improvement I think would be good, if you think it is useless just let me know.

Reply:
```suggestion
        return _(
```